engine/clients/elasticsearch/configure.py

```diff
@@ -18,26 +13,15 @@ class ElasticConfigurator(BaseConfigurator):
         Distance.COSINE: "cosine",
         Distance.DOT: "dot_product",
     }
+    # TODO: Add other types
     INDEX_TYPE_MAPPING = {
```

Comment:
which types we don't support at the moment?

Reply:
`keyword, text, float`. Added them right away.